Layout_style_v1.css

```diff
@@ -203,8 +250,8 @@ ul li:hover ul {
 .footer {
 	grid-column-end: span 2;
 	background-color: #B3E8FF;
-	width: 2010px;
-	Height: 200px;
+	width: 1990px;
+	Height: 70px;

```

Comment:
I changed the footer's height from 200px to 70px as it would be more suitable because I only need to add my Authenticity statement